src/components/PostDetails.tsx

```diff
@@ -1,100 +1,53 @@
 import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import * as commentsAction from '../features/comments';
```

Comment:
Ensure that the import path '../features/comments' correctly points to the file where the `commentsAction` is defined. If the path is incorrect, it will cause runtime errors.